src/rest_api/classes/generic_fields.clj

```diff
@@ -12,22 +13,22 @@
   (let [id-kw (first (filter #(= (name %) "id") (keys object)))
         role (namespace id-kw)]
     {:data (when-let [ghs ((keyword role "gene") object)]
-             (for [gh ghs :let  [gene ((keyword (str role ".gene") "gene") gh)]]
+             (for [gh ghs :let [gene ((keyword (str role ".gene") "gene") gh)]]
```

Comment:
Perhaps this shouldn't block merging, but minor style request to keep `:let` declarations in a `for` for it's own line, e.g:
```clojure
(for [x y
       :let [foo bar]]
   (do-stuff ...))
```